Challenge-4/Bot_2/src/agent.ts

```diff
@@ -0,0 +1,134 @@
+import {
+  HandleTransaction,
+  TransactionEvent,
+  ethers,
+  Finding,
+  getEthersProvider,
+} from "forta-agent";
+import {
+  BORROW_RATE,
+  SUPPLY,
+  SUPPLY_RATE,
+  USDC_TOKEN_ETH,
+  UTILIZATION,
+  WITHDRAW,
+} from "./constants";
+import Helper from "./helper";
+import CONFIG from "./agent.config";
+import { NetworkData } from "./types";
+import { NetworkManager } from "forta-agent-tools";
+import {
+  supplyFinding,
+  borrowFinding,
+  alertSupplyFinding,
+  alertBorrowFinding,
+} from "./findings";
+
+let configuratorProxy: string | undefined;
+let tokenAddress: string;
+let helper: Helper;
+const networkManager = new NetworkManager<NetworkData>(CONFIG);
+export function provideInitialize(provider: ethers.providers.Provider) {
+  return async function initialize() {
+    await networkManager.init(provider);
+    helper = new Helper(provider);
+  };
+}
+
+export function provideHandleTransaction(
+): HandleTransaction {
+  return async function HandleTransaction(tx: TransactionEvent) {
+    const finding: Finding[] = [];
```

Comment:
```suggestion
    const findings: Finding[] = [];
```
Suggestion to make it plural, as that would be more accurate.